wiki/en/Server-Troubleshooting.md

```diff
@@ -43,22 +43,14 @@ This should be your **public** IP address (find that with [Google](https://www.g
 
 You can set your server to listen on a custom port with the `--port` option. If you do this, you will need to tell people which port to connect on. They will need to append the port number to the address of your server in the format `[serverAddress]:[portNumber]`.
 
-### Is a private server a direct server without contact to the Jamulus Directory Server?
+### **_Headless Linux_**: I have created a server but it is joining a directory and I don't want it to.
 
-Yes. Please see [Server Types](Running-a-Server#server-types)
-
-### Can I convert my private server to being a public one?
-
-Yes. Simply check the "Make my Server Public" box in the server window and fill out the details. If you register your server, do not use port forwarding as it may prevent others using your server.
```

Comment:
Should be re-phrased and not deleted. Is it still possible to switch from registered to unregistered?

Reply:
I removed it because I really can't see anyone being confused about this. And it's going to be in the GUI now too.

Reply:
>  I really can't see anyone being confused about this

True... Probably this is on the wrong page then. Didn't fully test the new UI yet, so it might be evident with the new UI.

Reply:
Is it not covered elsewhere now?  Hard to tell with the changeset view...

Reply:
In so far as the UI for this will now be self-evident (and I think probably always was for this really) I don't think we need to document it because we don't document how to use the UI, only what it means to use it.